connected care: add features docs (#20303)
pingcap/docs: tidb-cloud/connected-ai-chat-in-im.md

---

> title: AI chat in IM

Reviewer: ```suggestion
title: "Connected: AI chat in IM"
```

> title: Connected: AI chat in IM

Reviewer 2: ```suggestion
title: AI chat in IM
```

> summary: Introduces detailed information about the AI chat in IM

Reviewer: ```suggestion
summary: Introduces detailed information about the AI chat in Instant Message (IM).
```

---

> # Connected: AI chat in IM

Reviewer: ```suggestion
# AI chat in IM
```

> The AI chat in Instant Message (IM) powered by PingCAP, is an offering to invite TiDB AI assistant chatbot to specific instant message (IM) channels for preliminary technical support and consultation. This service is based on Graph RAG built on top of TiDB Serverless Vector Storage.

Reviewer: ```suggestion
The AI chat in Instant Message (IM) powered by PingCAP, is an offering to invite TiDB AI assistant chatbot to specific IM channels for preliminary technical support and consultation. This service is based on Graph RAG (Retrieval-Augmented Generation) built on top of [TiDB Vector Search](/tidb-cloud/vector-search-overview.md).
```

## Limitation

> Currently, the ticket support feature for [Lark](https://www.larksuite.com/) is still in beta and is available upon request. If you are interested in trying this feature, contact TiDB Cloud support at <support@pingcap.com> or reach out to your Technical Account Manager (TAM).

Reviewer 2: ```suggestion
Currently, the ticket support feature for [Lark](https://www.larksuite.com/) is still in beta and is available upon request. If you are interested in trying this feature, contact TiDB Cloud support at <a href="mailto:support@pingcap.com">email</a> or reach out to your Technical Account Manager (TAM).
```

---
title: Connected: AI chat in IM
summary: Introduces detailed information about the AI chat in IM
---

# Connected: AI chat in IM

The AI chat in IM powered by PingCAP, is an offering to invite TiDB AI assistant chatbot to specific instant message (IM) channels for preliminary technical support and consultation. This service is based on Graph RAG built on top of TiDB Serverless Vector Storage.

> **Limitation:**
>

> > The Lark version is not fully ready yet. If you would like to take a try with the private preview release, please contact our support team at support@pingcap.com or your Technical Account Manager (TAM).

Reviewer 3: While the Slack version is fully ready, the Lark version is still in private preview. If you're interested in trying the current Lark release, please contact our support team at support@pingcap.com or reach out to your Technical Account Manager (TAM).
